tutorial.adoc

```diff
@@ -56,7 +56,8 @@ com.unblu.authentication.jwt.jwkUrl=https://application.example.com/api/jwk
 com.unblu.authentication.jwt.expectedIssuer=https://application.example.com
 com.unblu.authentication.jwt.expectedAudience=https://example.unblu.cloud
 com.unblu.authentication.jwt.useEncryption=false
-#com.unblu.authentication.jwt.encryptionKey=<private key, if encryption is enabled>
+#com.unblu.authentication.jwt.encryptionAlgorithm=<RSA or AES>
+#com.unblu.authentication.jwt.encryptionKey=<private key, if encryption is enabled and encryptionAlgorithm is RSA. When encryptionAlgorithm is set to AES, it must be base64 encoded byte array of 256 bits length.>
```

Comment:
wrong key?

Reply:
The key is correct. You can find related file [here](https://gitlab.uops.cloud/unblu/main/software-components/public-components/collaboration-server/collaboration-server/-/blob/23bba9620c82dd1eeb290ba7b86687a650784912/com.unblu.platform.server.core/src/com/unblu/platform/server/core/auth/configuration/jwt/JwtConfiguration.java)